docs/readthedocs/source/doc/Nano/Overview/pytorch_inference.md

```diff
@@ -368,3 +368,40 @@ with InferenceOptimizer.get_context(ipex_model, classifer):
     output = classifer(x) 
     assert torch.get_num_threads() == 4  # this line just to let you know Nano has provided thread control automatically : )
 ```
+
+## One-click Accleration Without Code Change
+```eval_rst
+.. note::
+    Neural Compressor >= 2.0 is needed for this function. You may call ``pip install --upgrade neural-compressor`` before using this functionality.
+```
+
+We also provides a no-code method for users to accelerate their pytorch inferencing workflow through neural coder. Users could call
+
+```bash
+python -m neural_coder -o <acceleration_name> example.py
+```
+
+For `example.py`, it could be a common pytorch inference script without any code changes needed. For `<acceleration_name>`, please check following table.
+
+| Optimization Set | <acceleration_name> | 
```

Comment:
There seems to be a problem with the rendered effect：
![image](https://user-images.githubusercontent.com/105281011/210685218-dd52427e-bf60-4560-86b9-2242bc6f89bd.png)
no `acceleration_name` now